src/app/modules/library/personal-library/personal-library.component.ts

```diff
@@ -80,6 +115,69 @@ export class PersonalLibraryComponent extends LibraryComponent {
   protected getDetailsComponent(): any {
     return PersonalLibraryDetailsComponent;
   }
+
+  public filterUpdated(filterValues: ProjectFilterValues = null): void {
+    super.filterUpdated(filterValues);
+    this.filteredProjects = this.filteredProjects.filter(
+      (project) => project.tags.includes('archived') == this.showArchived
+    );
+  }
+
+  protected switchActiveArchivedView(): void {
+    this.filterUpdated();
+    this.unselectAllProjects();
+  }
+
+  pageChange(event?: PageEvent, scroll?: boolean): void {
+    super.pageChange(event, scroll);
+    this.unselectAllProjects();
+  }
+
+  protected projectSelectionChanged(event: any): void {
+    if (event.checked) {
```

Comment:
Change field name from "checked" to "selected" to match "selection"?
```suggestion
    if (event.selected) {
```